tests/data/power_flow/asym-line/input.json

```
@@ -75,35 +75,36 @@
           "x_nb": 0.7339,
           "x_nc": 0.7559,
           "x_nn": 0.8316,
-          "c0" : 0.32,
-          "c1" : 0.54, 
+          "c0" : 0.32e-9,
+          "c1" : 0.54e-9, 
           "i_n" : 115.0
         }
     ],
-    "asym_load": [
+    "source": [
       {
-        "id": 6,
-        "node": 1,
-        "status": 1,
-        "type": 1,
-        "p_specified": [1000.0, 1000.0, 1000.0],
-        "q_specified": [1000.0, 1000.0, 1000.0]
-      },
+        "id": 6, 
+        "node": 1, 
+        "status": 1, 
+        "u_ref": 10000,
```

```suggestion
        "u_ref": 1.0,
```
This quantity is in p.u.